src/agent/onefuzz-agent/src/managed/cmd.rs

```diff
@@ -23,6 +39,35 @@ pub async fn run(args: &clap::ArgMatches<'_>) -> Result<()> {
     result
 }
 
+// Periodically check available system memory.
+//
+// If available memory drops below the minimum, exit informatively.
+async fn out_of_memory(min_bytes: u64) -> Result<OutOfMemory> {
```

Comment:
I could have use the private named constant that we always pass here. I only parameterized this check to make it clear how we'd make  it configurable (and thread the exact limit through to the final error message).

Reply:
add that as comment in the code ?

Reply:
IMO the actual usage is clear enough, I was more just noting it here for discussion's sake.

Do we think we'll want to make it tunable (i.e. per-node or task) in the future? If not, I can just simplify it by making it NOT parameterized.

Reply:
(just added a comment anyway)

Reply:
I'm ok with leaving it tunable.